content/python/concepts/errors/errors.md

```diff
@@ -37,7 +37,7 @@ Even if a statement or expression is syntactically correct, it may cause an erro
 
```

Comment:
Also, on line # 30, can we do the following:

1. Switch around "function" and "print()" with each other. (i.e., ~"... at the function `print()`, since a ..."~ --> "... at the `print()` function, since a ...")
2. Add a hyperlink to "`print()`" by linking out to the following Docs term entry: https://www.codecademy.com/resources/docs/python/built-in-functions/print